public/graph.js

```diff
@@ -17,6 +17,7 @@ class GraphController {
             isYLimitSentManuallySet = false,
             isYLimitFailedManuallySet = false,
             dayDateFormat = d3.timeFormat("%Y-%m-%d"),
+            dayTimeFormat = d3.timeFormat("%H:%M %p"),
```

Comment:
please investigate what's causing this.
![image](https://user-images.githubusercontent.com/23135322/80790666-20cfca80-8b98-11ea-830e-88d4233d998e.png)

Reply:
I will